net/tcp/impl/posix.h

```diff
@@ -274,6 +275,9 @@ class Connection : public SocketHandle {
   }
 
   inline Connection& BlockingWrite(const void* buffer, size_t write_length, bool more) {
+#ifdef BRICKS_APPLE
+    static_cast<void>(more);  // Supress the 'unused parameter' warning on Apple.
```

Comment:
I would make it without any ifdefs - looks like it is not used not only in BRICKS_APPLE case.